--- ezmlm-web-3.3.1.org/ezmlm-web.cgi	2008-10-15 22:17:16.000000000 +0000
+++ ezmlm-web-3.3.1/ezmlm-web.cgi	2018-08-01 12:06:55.204507858 +0000
@@ -237,13 +237,13 @@
 $HTML_TITLE = '' unless defined($HTML_TITLE);
 
 # check HTML_LINKS
-@HTML_LINKS = () unless defined(@HTML_LINKS);
+@HTML_LINKS = () unless (@HTML_LINKS);
 
 # check DEFAULT_INTERFACE_TYPE
 $DEFAULT_INTERFACE_TYPE = 'normal' unless defined($DEFAULT_INTERFACE_TYPE);
 
 # check possible blacklist of interface options (since v3.3)
-@INTERFACE_OPTIONS_BLACKLIST = () unless defined(@INTERFACE_OPTIONS_BLACKLIST);
+@INTERFACE_OPTIONS_BLACKLIST = () unless (@INTERFACE_OPTIONS_BLACKLIST);
 
 # check the configured detault location of gnupg keyrings
 $GPG_KEYRING_DEFAULT_LOCATION = ".gnupg"
