diff -ruN streamhtmlparser/Makefile.am streamhtmlparser-patched/Makefile.am
--- streamhtmlparser/Makefile.am	2021-04-08 19:04:03.394091174 +0200
+++ streamhtmlparser-patched/Makefile.am	2021-04-07 21:09:28.829112439 +0200
@@ -4,7 +4,7 @@
 ACLOCAL_AMFLAGS = -I m4
 
 # This is so we can #include <streamhtmlparser/foo>
-AM_CPPFLAGS = -I$(top_srcdir)/src
+AM_CPPFLAGS = -I$(top_srcdir)/src -fPIC
 
 # These are good warnings to turn on by default
 if GCC
